knative-operator/cmd/manager/main.go

```diff
@@ -135,6 +135,8 @@ func main() {
 
 	decoder := admission.NewDecoder(mgr.GetScheme())
 
+	// This call adds the server to the manager as a Runnable
+	_ = mgr.GetWebhookServer()
```

Comment:
```
func (cm *controllerManager) GetWebhookServer() webhook.Server {
	cm.webhookServerOnce.Do(func() {
		if cm.webhookServer == nil {
			panic("webhook should not be nil")
		}
		if err := cm.Add(cm.webhookServer); err != nil {
			panic(fmt.Sprintf("unable to add webhook server to the controller manager: %s", err))
		}
	})
	return cm.webhookServer
}
```

Without this webhook server is not available.

Reply:
We used to have that: https://github.com/openshift-knative/serverless-operator/blob/release-1.26/knative-operator/cmd/manager/main.go#L105.